src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -4867,31 +4867,168 @@ internal abstract partial class TypeOperationExpression : Operation, ITypeOperat
     /// change it in the future.
     /// </remarks>
-    internal sealed partial class TypeParameterObjectCreationExpression : Operation, ITypeParameterObjectCreationExpression
+    internal abstract partial class BaseDynamicObjectCreationExpression : Operation, IHasDynamicArgumentsExpression, IDynamicObjectCreationExpression
     {
-        public TypeParameterObjectCreationExpression(SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+        public BaseDynamicObjectCreationExpression(string name, ImmutableArray<ISymbol> applicableSymbols, ImmutableArray<string> argumentNames, ImmutableArray<RefKind> argumentRefKinds, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
             base(OperationKind.TypeParameterObjectCreationExpression, syntax, type, constantValue)
         {
+            Name = name;
+            ApplicableSymbols = applicableSymbols;
+            ArgumentNames = argumentNames;
+            ArgumentRefKinds = argumentRefKinds;
         }
         public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield break;
+                yield return Initializer;
             }
         }
+        /// <summary>
+        /// Name of the dynamically invoked member.
+        /// </summary>
+        public string Name { get; }
+        /// <summary>
+        /// List of applicable symbols that are dynamically bound to the <see cref="Name"/>.
+        /// </summary>
+        public ImmutableArray<ISymbol> ApplicableSymbols { get; }
+        /// <summary>
+        /// Dynamically bound arguments, excluding the instance argument.
+        /// </summary>
+        public abstract ImmutableArray<IOperation> Arguments { get; }
+        /// <summary>
+        /// Optional argument names for named arguments.
+        /// </summary>
+        public ImmutableArray<string> ArgumentNames { get; }
+        /// <summary>
+        /// Optional argument ref kinds.
+        /// </summary>
+        public ImmutableArray<RefKind> ArgumentRefKinds { get; }
+        /// <summary>
+        /// Object or collection initializer, if any.
+        /// </summary>
+        public abstract IObjectOrCollectionInitializerExpression Initializer { get; }
         public override void Accept(OperationVisitor visitor)
         {
-            visitor.VisitTypeParameterObjectCreationExpression(this);
+            visitor.VisitDynamicObjectCreationExpression(this);
         }
         public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
         {
-            return visitor.VisitTypeParameterObjectCreationExpression(this, argument);
+            return visitor.VisitDynamicObjectCreationExpression(this, argument);
+        }
+    }
+
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
```

Comment:
Looks like this comment was copied from the interface unnecessarily. Same for other classes here.